### DISCUSSION TITLE
Can someone explain where I am going wrong, thanks!

### DISCUSSION THREAD

**Discussion by TheCollecta** [community/community#90524] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

my repository knd heads here... https://thecollecta.github.io/knd/ 

all my pages appear to be linked correctly and work well through vscode etc. have index.html in root folder and a subfolder called paged which has all other pages. however in github the pages dont link and i get 404 or the html appears but the css is not linked? 

thanks in advance, very new to this so go steady! ha

**Comment by saibushifu** [comment]
Look on the dir in GH interface and use it.

![image](https://github.com/community/community/assets/156952871/719a32c4-3faf-4736-bd68-4cc93f5dad33)

![image](https://github.com/community/community/assets/156952871/a7a92bee-7064-420d-b70a-ce2a6c64f369)

**Reply by TheCollecta** [reply]
thats the wrong one sorry its the KND one

**Reply by saibushifu** [reply]
![image](https://github.com/community/community/assets/156952871/b43a8748-6a05-4196-82d3-92a00abd527f)

**Reply by TheCollecta** [reply]
so i need to use /knd/paged/about.html?

**Reply by saibushifu** [reply]
CSS is here:
https://thecollecta.github.io/knd/style.css

**Reply by TheCollecta** [reply]
oh okay thank you lemme try sort this and ill update

**Reply by saibushifu** [reply, marked as answer]
Your page links doesn't contain .../knd/...

This's main mistake

**Reply by TheCollecta** [reply]
ive updated all these files now with /knd/paged/about.html for eg. but still nothing

**Reply by TheCollecta** [reply]
All sorted now! Just need to do the images! Thank you all!
